src/routes/ListPlans.js

```diff
@@ -0,0 +1,142 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+import * as actions from '../actions';
+
+import {
+    Main,
+    PageHeader, PageHeaderTitle
+} from '@red-hat-insights/insights-frontend-components';
+
+import {
+    Grid, GridItem,
+    Card, CardHeader, CardBody,
+    Progress, ProgressMeasureLocation,
+    Stack, StackItem,
+    Switch,
+    Level, LevelItem
+} from '@patternfly/react-core';
+
+import './ListPlans.scss';
+
+class ListPlans extends Component {
```

Comment:
This is more "RemediationDetails" page rather than a list, right?

Reply:
Correct. We can update the name.